packages/migrate/src/plugins/diagnostics-autofix.plugin.ts

```diff
@@ -17,42 +17,43 @@ export default class DiagnosticAutofixPlugin extends Plugin {
 
     this.logger?.debug(`Plugin 'DiagnosticAutofix' run on ${params.fileName}`);
 
+    const allFixedFiles: Set<string> = new Set();
```

Comment:
You can just use array directly:
```
const allFixedFiles: string[] = [];
...
allFixedFiles.push(fixedFile.fileName);
...
return allFixedFiles;
```

Reply:
Use Set to deduplicate possible duplicate names of the same files that may have been changed by different diagnostics.

Reply:
👍